strands_navigation_msgs/msg/Edge.msg

```diff
@@ -1,2 +1,9 @@
+string edge_id
 string node
 string action
+float64 top_vel
+string map_2d
+strands_navigation_msgs/DynClassLoaderDef[] monitor_states
+strands_navigation_msgs/DynClassLoaderDef[] recovery_state_machines
+strands_navigation_msgs/DynClassLoaderDef[] ui_helpers
+
```

Comment:
you need another one for the recovery_nav: there are recovery state machines for bumper, stuck on carpet etc., which are associated with monitors, and then a recovery state machine for navigation itself (the one with backtrack etc., which needs to be defined separately).